### DISCUSSION TITLE
Mirage v17 Support

### DISCUSSION THREAD

**Discussion by jmillerob** [primefaces/community#904] [PrimeNG Templates] [answered]
I see other themes have received support for v17, but Mirage has not. Has this theme become legacy?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Mirage has become a legacy template and will not receive an official update. However, we're preparing a manual update guide for the users. The guide will be available in upcoming weeks in `guides` section in primeng.org.

**Comment by jmillerob** [comment]
When will the upgrade guide be published? When I go to https://primeng.org/guides I do not see it anywhere.
